pkg/commands/app.go

```diff
@@ -520,6 +520,13 @@ func NewConvertCommand(globalFlags *flag.GlobalFlagGroup) *cobra.Command {
 		ReportFlagGroup: flag.NewReportFlagGroup(),
 	}
 
+	// To display the summary table, we need to enable scanners (to build columns).
+	// We can't get scanner information from the report (we don't include empty licenses and secrets in the report).
+	// So we need to ask the user to configure scanners (if needed).
+	convertFlags.ScanFlagGroup.Scanners = flag.ScannersFlag.Clone()
+	convertFlags.ScanFlagGroup.Scanners.Default = nil // disable default scanners
+	convertFlags.ScanFlagGroup.Scanners.Usage = "List of scanners included when generating the json report. Used only for rendering the summary table."
```

Comment:
I thought we could just enable all scanners. But I think we need to use the same format(logic) for `convert` and other subcommands.
So i enabled `--scanners` that users can get summary table as for `table` format.